lib/membrane/rtp/session/receive_bin.ex

```diff
@@ -54,6 +54,22 @@ defmodule Membrane.RTP.Session.ReceiveBin do
                 Function generating receiver SSRCs. Default one generates random SSRC
                 that is not in `local_ssrcs` nor `remote_ssrcs`.
                 """
+              ],
+              secure?: [
+                type: :boolean,
+                default: false,
+                description: """
+                Specifies whether to use SRTP.
+                Requires adding [srtp](https://github.com/membraneframework/elixir_srtp) dependency to work.
```

Comment:
```suggestion
                Requires adding [`:srtp`](https://github.com/membraneframework/elixir_srtp) dependency to work.
```
BTW this name seems kind of wierd, maybe we should add some prefix? E.g. `:ex_libsrtp`